sites/svelte.dev/src/routes/auth/login/+server.js

```diff
@@ -1,16 +1,15 @@
-import { stringify } from 'querystring';
 import { redirect } from '@sveltejs/kit';
-import { oauth, client_id } from '../_config.js';
+import { client_id, oauth } from '../_config.js';
 
 export const GET = client_id
 	? ({ url }) => {
 			const Location =
 				`${oauth}/authorize?` +
-				stringify({
+				new URLSearchParams({
```

Comment:
Why replace `stringify`? It's a built-in node module so we could rely on it. (same for another place)

Reply:
That's the problem: It's a `node` module. I kept getting errors about it, so replaced it with URLSearchParams, which is built into node and edge workers

Reply:
ah right of course!